lib/ansible/plugins/connection/psrp.py

```diff
@@ -410,10 +410,12 @@ def _connect(self):
 
     def reset(self):
         if not self._connected:
```

Comment:
Why isn't this 
```suggestion
        if self._connected:
            return self
```
The guard expression style having less indented blocks accommodating better readability